#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()
gen.add("angle", double_t, 0, "angle", 0, 0.0, pi)
gen.add("angle_thr", double_t, 0, "angle threshold", 10.0 / 180.0 * pi, 0.0, pi)

exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "PlaneRejector"))
